File: javascript/busqLineal.js
let array = [10];
let numBuscado = Math.floor(Math.random()*30 + 1)
let posicion = 0;
let bandera = false;

for (let i= 0;i<10;i++){
    array[i]= Math.floor(Math.random()*30 + 1)
}

for (let i= 0; i<10;i++){
    if(array[i]==numBuscado){
        bandera = true;
        posicion = i+1;
    }
}

if(bandera==true){
    console.log("El numero "+numBuscado+", fue encontrado en la posicion "+posicion+" del array");
}
else{
    console.log("El numero "+numBuscado+", no fue encontrado en el array");
}

File: c#/busqLineal.cs
using System;

class BusqLineal
{
    public static void Main(string[] args)
    {
        Random rand = new Random();
        int[] numeros = new int[10];
        int numeroBus = rand.Next(0, 20);
        int posic = 0;
        bool bandera = false;

        for (int i = 0; i < 10; i++)
        {
            numeros[i] = rand.Next(0, 20);
        }

        for (int i = 0; i < 10; i++)
        {
            if (numeros[i] == numeroBus)
            {
                bandera = true;
                posic = i + 1;
            }
        }

        if (bandera == true)
        {
            Console.WriteLine("El numero " + numeroBus + ", se encuentra en la posicion " + posic + " del array");
        }
        else
        {
            Console.WriteLine("El numero " + numeroBus + " no se encuentra dentro del array");
        }

    }
}
